pybullet_toolkit/robots/ur5_rg2.py

```diff
@@ -165,22 +193,19 @@ def _sendPositionCommand(self, commands):
     num_motors = len(self.arm_joint_indices)
     pb.setJointMotorControlArray(self.id, self.arm_joint_indices, pb.POSITION_CONTROL, commands,
                                  [0.]*num_motors, self.max_forces[:-2], [0.01]*num_motors, [1.0]*num_motors)
-    if self.gripper_closed:
-      self._sendGripperCloseCommand()
-    else:
-      self._sendGripperOpenCommand()
 
   def _sendGripperCloseCommand(self):
-    pb.setJointMotorControlArray(self.id, [10,11], pb.VELOCITY_CONTROL, targetVelocities=[1.0, 1.0], forces=self.gripper_close_force)
+    pb.setJointMotorControl2(self.id, 10, pb.POSITION_CONTROL, targetPosition=0.037, force=self.gripper_close_force[0])
```

Comment:
Any reason for this change? The array version is supposed to be faster according to the pybullet docs